exporter/signalfxexporter/factory_test.go

```diff
@@ -833,7 +833,8 @@ func TestDefaultExcludes_translated(t *testing.T) {
 	cfg := f.CreateDefaultConfig().(*Config)
 	setDefaultExcludes(cfg)
 
-	converter, err := translation.NewMetricsConverter(zap.NewNop(), testGetTranslator(t), cfg.ExcludeMetrics, cfg.IncludeMetrics, "")
+	// disable translations in order to avoid producing the default cpu.utilization metric from cpu.utilization_per_core
+	converter, err := translation.NewMetricsConverter(zap.NewNop(), nil, cfg.ExcludeMetrics, cfg.IncludeMetrics, "")
```

Comment:
Since this test ensures default excludes work as expected on translated metrics, we shouldn't disable translations all together. I would add `cpu.utilization` to `config.ExcludeMetrics`.

Reply:
@asuresh4 do you mean to add `cpu.utilization` particularly for this test? 

Wouldn't it be better to keep it enabled and change the test to expect one `cpu.utilization` datapoint left as a result of the translations?

Reply:
Yes, I was suggesting to add `cpu.utilization` to the disabled list. That said, what you're suggesting would work as well. I'm ok with either as long as we don't disable translations for this test.